core/command_line_parser.cpp

```diff
@@ -13,15 +13,15 @@ struct CommandLineOption::ArgumentChecker {
 struct CommandLineOption::FunctionChecker : public CommandLineOption::ArgumentChecker {
 	CheckFunction function;
 
-	bool check(const String &p_arg) const override {
```

Comment:
Why?..

Reply:
If you ask about `override` removal specifically, this is because Godot 3.x fails to compile with `werorr=yes` with inconsistent override warning in Godot 3.x. When all of the code is going to be ported to Godot 4.0, it will make sense to add `override` keywords.